plugin/kubernetes/parse_test.go

```diff
@@ -14,17 +14,17 @@ func TestParseRequest(t *testing.T) {
 		expected string // output from r.String()
 	}{
 		// valid SRV request
-		{"_http._tcp.webs.mynamespace.svc.inter.webs.test.", "http.tcp..webs.mynamespace.svc"},
+		{"_http._tcp.webs.mynamespace.svc.inter.webs.tests.", "http.tcp..webs.mynamespace.svc"},
```

Comment:
Why are these changes necessary?  `test` => `tests`

Reply:
Yes, this is a typo before. The query zone is `inter.webs.tests`,but domain use `*.inter.webs.test`.
 https://github.com/coredns/coredns/blob/8aa55c5ff2f30b2d4b4c5faad5e0a465648f9736/plugin/kubernetes/parse_test.go#L62